pkgs/development/tools/analysis/radare2/default.nix

```diff
@@ -79,6 +79,12 @@ stdenv.mkDerivation rec {
    "-Dr2_gittap=${version}"
   ];
 
+  # TODO: remove when upstream fixes the issue
+  # https://github.com/radareorg/radare2/issues/22793
+  NIX_CFLAGS_COMPILE = lib.optionals stdenv.isDarwin [
+     "-DTHREAD_CONVERT_THREAD_STATE_TO_SELF=1"
+  ];
```

Comment:
I think `NIX_CFLAGS_COMPILE` should be a string:
```suggestion
  env.NIX_CFLAGS_COMPILE = toString (lib.optionals stdenv.isDarwin [
     "-DTHREAD_CONVERT_THREAD_STATE_TO_SELF=1"
  ]);
```

Reply:
Yeah I am too used to the old days style, sorry.